lib/bulkrax.rb

```diff
@@ -33,6 +34,10 @@ class << self
     self.removed_image_path = Bulkrax::Engine.root.join('spec', 'fixtures', 'removed.png').to_s
     self.server_name = 'bulkrax@example.com'
 
+    self.base_path = lambda do |type|
+      ENV['HYKU_MULTITENANT'] ? File.join(Bulkrax.send("#{type}_path"), Site.instance.account.name) : Bulkrax.send("#{type}_path")
+    end
+
```

Comment:
I don't think this should be here as a lambda, I think it would be better suited to existing as a regular method in the application_parser. Also that would be more readable.

Reply:
@sephirothkod I tried to put a method in application parser first, but it didn't work there. I'll try again tomorrow though.